src/ucp/api/ucp.h

```diff
@@ -3292,6 +3292,51 @@ ucp_atomic_fetch_nb(ucp_ep_h ep, ucp_atomic_fetch_op_t opcode,
                     ucp_send_callback_t cb);
 
 
+/**
+ * @ingroup UCP_COMM
+ * @brief Post an atomic fetch operation.
+ *
+ * This routine will post an atomic fetch operation to remote memory.
+ * The remote value is described by the combination of the remote
+ * memory address @a remote_addr and the @ref ucp_rkey_h "remote memory handle"
+ * @a rkey.
+ * The routine is non-blocking and therefore returns immediately. However the
+ * actual atomic operation may be delayed. The atomic operation is not considered complete
+ * until the values in remote and local memory are completed.
+ *
+ * @note The user should not modify any part of the @a buffer or @a result after
+ *       this operation is called, until the operation completes.
+ * @note Only contig(32) or contig(64) datatypes are supported in
+ *       @a param->datatype, see @ref ucp_dt_make_contig
+ *
+ * @param [in] ep          UCP endpoint.
+ * @param [in] opcode      One of @ref ucp_atomic_fetch_op_t.
+ * @param [in] buffer      Address of operand for atomic operation. In the
+ *                         case of CSWAP this is the conditional for the swap.
```

Comment:
`Address of operand for atomic operation. For @ref CSWAP operation, this is the
value the remote memory buffer is compared with.`

Reply:
fixed